risc0/circuit/recursion/src/control_id.rs

```diff
@@ -39,15 +37,12 @@ pub const ALLOWED_CONTROL_IDS: &[Digest] = &[
     digest!("c607b103592326302802cf1ff1c6ee5dab7f112e85c6106b4085e55c46535f1d"), // recursion lift_19.zkr
     digest!("deca5917ab949d3769a17415d4b5753d0a49f818af600815ad92c85067322217"), // recursion lift_20.zkr
     digest!("cc4245191072ea38989b65721112584cd548385f0eda402d76c70e0e718e0f73"), // recursion lift_21.zkr
-    digest!("dd6cd51e766f484d625df405e74e11067283c1269dbdae21bc6a031b1daa952e"), // recursion lift_22.zkr
-    digest!("4726ab732cc10d26e233aa6c82a32001f2d83a1878e5da7531017b737054dc6d"), // recursion lift_23.zkr
-    digest!("8ea32467c6445926da67365554e3b64be92a5076d071ff0632f45b294787253f"), // recursion lift_24.zkr
```

Comment:
Seems like we still need these

Reply:
See https://github.com/risc0/risc0/pull/2276#issuecomment-2313212347 my approach here.